tests/test_main.py

```diff
@@ -1,22 +1,40 @@
 """
-Unit tests for testing the smart home app.
+Unit tests for the Smart Home Automation App.
 """
 
 from main import SmartHomeApp
 import hashlib
 
 
-class TestSuccess:
+class TestSmartHomeApp:
     def setup_method(self):
-        """
-        pytest method ran before each pytest to set it up
-        """
+        """Pytest method run before each test."""
         self.app = SmartHomeApp()
 
+    # ----- Test Utility Functions -----
+
     def test_hash_password(self):
+        """Tests the hashed password is returned properly"""
         password = "securepassword123"
```

Comment:
```suggestion
        password = "SecurePassword123!"
```
else the password isn't actually valid

Reply:
Resolved in [07facbb](https://github.com/rowan04/L5-SE-CW2-QualityPlan/pull/12/commits/07facbbca4c3f347667c76b0c2b8d527edb29489)